apiv6/generate_client.sh

```diff
@@ -56,33 +83,24 @@ function cleanup_non_needed_files() {
 
     generated_content_dir="$1"
 
+    rm "$generated_content_dir/.travis.yml"
+    rm "$generated_content_dir/git_push.sh"
     rm "$generated_content_dir/go.mod"
     rm "$generated_content_dir/go.sum"
     rm "$generated_content_dir/README.md"
-    rm -rf "$generated_content_dir/docs"
     rm -rf "$generated_content_dir/api"
+    rm -rf "$generated_content_dir/docs"
 
     echo "Done"
 }
 
-checkPrerequsites
-
-# GENERATE CLOUDEXPORT
-cloudexport_package="cloudexport"
-cloudexport_spec="cloud_export.openapi.yaml"
-
-cloudexport_client_output_dir="kentikapi/cloudexport"
-
-generate_golang_client_from_openapi3_spec "$cloudexport_spec" "$cloudexport_package" "$cloudexport_client_output_dir"
-change_ownership_to_current_user "$cloudexport_client_output_dir"
-cleanup_non_needed_files "$cloudexport_client_output_dir"
-
-# GENERATE SYNTHETICS
-synthetics_package="synthetics"
-synthetics_spec="synthetics.openapi.yaml"
+function stage() {
+    BOLD_BLUE="\e[1m\e[34m"
+    RESET="\e[0m"
+    msg="$1"
```

Comment:
The color tricks do not always work in my environment. Most of the time, it just spews control codes.

Reply:
Created bug: https://clkentik.atlassian.net/browse/KNTK-288